omod/src/main/webapp/pages/patientRegistration/children.gsp

```diff
@@ -0,0 +1,81 @@
+<%
+    ui.decorateWith("appui", "standardEmrPage")
+    ui.includeJavascript("uicommons", "datatables/jquery.dataTables.min.js")
+
+%>
+
+${ ui.includeFragment("coreapps", "patientHeader", [ patient: patient ]) }
+
+<script type="text/javascript">
+    var breadcrumbs = [
+        { icon: "icon-home", link: '/' + OPENMRS_CONTEXT_PATH + '/index.htm' },
+        { label: "${ ui.escapeJs(ui.format(patient.patient)) }" , link: '${ui.pageLink("pihcore", "router/programDashboard", ["patientId": patient.id])}'},
+        { label: "${ ui.message("pihcore.encounterList") }" , link: '${ui.pageLink("pihcore", "patient/encounterList", ["patientId": patient.id])}'}
```

Comment:
Will need to clean up this reference - I assume this came from the page you copied to create this?